src/schema/__tests__/formattedDaySchedules.test.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  FormattedDaySchedules,
+  DaySchedule,
+} from "../types/formattedDaySchedules"
+
+const formatDaySchedules = FormattedDaySchedules.resolve
+
+describe("daySchedules", () => {
+  it("collapses times duplicated across days", () => {
+    const daySchedules = [
+      {
+        day_of_week: "Monday",
+        start_time: 37800,
+        end_time: 68400,
+      },
+      {
+        day_of_week: "Tuesday",
+        start_time: 37800,
+        end_time: 68400,
+      },
+      {
+        day_of_week: "Wednesday",
+        start_time: 37800,
+        end_time: 68400,
+      },
+      {
+        day_of_week: "Thursday",
+        start_time: 37800,
+        end_time: 68400,
+      },
+      {
+        day_of_week: "Friday",
+        start_time: 37800,
+        end_time: 68400,
+      },
+      {
+        day_of_week: "Saturday",
+        start_time: 37800,
+        end_time: 68400,
+      },
+    ]
+    expect(formatDaySchedules(daySchedules as Array<DaySchedule>)).toEqual([
+      {
+        days: "Monday–Saturday",
+        hours: "10:30am–7pm",
+      },
+    ])
+  })
+
+  it("handles multiple time segments in a single day", () => {
+    const daySchedules = [
+      {
+        start_time: 39600,
+        end_time: 50400,
+        day_of_week: "Sunday",
+      },
+      {
+        start_time: 61200,
+        end_time: 72000,
+        day_of_week: "Sunday",
+      },
+      {
+        start_time: 39600,
+        end_time: 50400,
+        day_of_week: "Monday",
+      },
+      {
+        start_time: 64800,
+        end_time: 72000,
+        day_of_week: "Monday",
+      },
+      {
+        start_time: 39600,
+        end_time: 50400,
+        day_of_week: "Wednesday",
+      },
+      {
+        start_time: 64800,
+        end_time: 72000,
+        day_of_week: "Wednesday",
+      },
+      {
+        start_time: 39600,
+        end_time: 50400,
+        day_of_week: "Thursday",
+      },
+      {
+        start_time: 64800,
+        end_time: 72000,
+        day_of_week: "Thursday",
+      },
+      {
+        start_time: 39600,
+        end_time: 50400,
+        day_of_week: "Friday",
+      },
+      {
+        start_time: 64800,
+        end_time: 72000,
+        day_of_week: "Friday",
+      },
+      {
+        start_time: 39600,
+        end_time: 50400,
+        day_of_week: "Saturday",
+      },
+      {
+        start_time: 64800,
+        end_time: 72000,
+        day_of_week: "Saturday",
+      },
+    ]
+    expect(formatDaySchedules(daySchedules as Array<DaySchedule>)).toEqual([
+      {
+        days: "Monday, Wednesday–Saturday",
+        hours: "11am–2pm, 6pm–8pm",
+      },
+      {
+        days: "Sunday",
+        hours: "11am–2pm, 5pm–8pm",
+      },
+    ])
+  })
+})
```

Comment:
I was comparing these to [these specs in Force](https://github.com/artsy/force/blob/33ece666e9df1cf32f9c6cbe7914811815a860cc/src/desktop/test/models/partner_show.coffee#L278) — one thing that's not clear to me is if/when we should expect to see "Closed" in the formatted output.

Seems from [this line](https://github.com/artsy/metaphysics/pull/1397/files#diff-4a5993f0c9dcea4bf4fec291505453d7R49) like there is such a condition, and I thought maybe it would apply to Tuesday in the above example?

A spec that clarifies the "Closed" behavior might generally be helpful here.

Reply:
Per our discussion at standup, I've updated this so that "Closed" days are no longer filtered from the result. Thanks for calling this out!